memoized_lcsh.txt

```diff
@@ -0,0 +1 @@
+{"Sailboats"=>"sh2011004125", "Periodical cicada"=>"sh85099887", "Saguenay River (Quebec)"=>"sh86006143", "Social stratification"=>"sh99000315", "Originality"=>"sh85095646", "Sailing"=>"sh91001077", "Marsden"=>"sh99014562", " Sam (Fictitious character)"=>"sh2008000533", "Molds (Fungi)"=>"sh85086566", "Lindens"=>"sh85077148", "Statues"=>"sh2003001401", "Test family"=>"sh85080045"}
```

Comment:
This isn't really a JSON file, it's just a ruby hash that we call `eval` on so I decided to just use the `txt` extension since it's versatile.